turingschool/ruby-exercises
# Re-do video walk-through of setup and passing the `data-types/strings/strings.rb` test file

**Don't merge until this video is up/viewable on Youtube: [https://youtu.be/aeAkLxr5diE](https://youtu.be/aeAkLxr5diE). It's uploading (slowly-ish) right now.**

Broadly, I wanted to replace the video walk-through that I'd made long ago, with a new one that is more up-to-date, etc. 

Broadly, I've reduced complexity of the set-up and tests, and created an improved walk-through that covers _so much more_ than just making the tests pass.

Covers 
- how to effectively move around your terminal and editor
- good workflows/processes
- how to google for things
- how to read the docs, what to attend to/ignore
- how to "test" ideas you get from the docs
- how to organize _physical, hand-written notes_ to maximize engagement/retention with the material
- Notes on how to pace yourself
- much more

I first made this video two weeks ago (recorded it), really wanting to get it in front of the #2006-be as they're gearing up for Turing.

I _really_ want students to get in the habits and processes I've outlined in this video, and I wanted it to go to them before end of memorial day weekend, so... merging this in tonight. 

This sits on top of the work done in #68 so I'm going to merge in both, make sure all looks good, then pass to the upcoming cohort so they can start working on it. 

I could go into far more detail about the exact specifics of every LOC change in the PR, but... only if anyone wants it! Otherwise, I'll save my essays for another day. 

![writing](https://media0.giphy.com/media/nGtOFccLzujug/giphy.gif?cid=e1bb72ff935f716edd24b5f324d9cc3b55ba0fe035da5c6f&rid=giphy.gif)
